src/components/album/AlbumChat.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
+import io from 'socket.io-client';
+
+const AlbumChat = ({ apiUrl }) => {
+  useEffect(() => {
+    const socket = io('http://localhost:8000');
```

Comment:
Just like the HTTP API client in the frontend, we should use an environment variable so that this points to localhost when running locally and [https://musicstore.onrender.com](https://musicstore.onrender.com) when running in production (eg: the API uses `REACT_APP_API_BASE_PATH`, here we could use `REACT_APP_SOCKET_BASE_PATH`)

Reply:
.env: REACT_APP_SOCKET_BASE_PATH=http://localhost:8000
REACT_APP_SOCKET_BASE_PATH= https://musicstore.onrender.com

<img width="654" alt="screen" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/95896159/5ce1123f-47f5-4304-9709-d708dc5df031">